### ISSUE TITLE
قلب  (Qalb) output is left-to-right arabic script

### ISSUE THREAD

**Issue by razzius** [radian-software/riju#143]
Hi, I'd never heard of قلب before looking around riju; thanks for adding it!

There is a problem with the output, however; the arabic letters go from left to right and are not connected, as arabic usually is:

<img width="854" alt="image" src="https://user-images.githubusercontent.com/2244895/145352479-c554753a-7324-4503-8560-9a830332fbfc.png">

The first word, مرحبا, is displayed essentially "اب ح رم"; it should look how it looks on the source pane side (and be the word the furthest to the right, since it's the start of the right-to-left line).

**Comment by raxod502** [comment]
Yeah, that's fair, it would be a bit tricky to fix this since we don't fully control the terminal output (that's from xterm.js), but worth a try.
